sdks/java/container/license_scripts/pull_licenses_java.py

```diff
@@ -39,7 +39,7 @@
 from urllib.request import urlopen, URLError, HTTPError
 
 SOURCE_CODE_REQUIRED_LICENSES = ['lgpl', 'glp', 'cddl', 'mpl']
-RETRY_NUM = 3
+RETRY_NUM = 9
```

Comment:
There is an exponential weight on L47. With 9 retries it will wait for a long time. I do not know what is the multiplier, but even with a multiplier of 1, this will be almost 10 minutes wait before the last retry.

Should we change retry number? Or the exponential multiplier?

Reply:
I suggest we use wait_fixed(5) instead of the wait_exponential. Since we were only setting retries to 3 before it should have been waiting:
2s, 4s, 8s.

wait_fixed(5) will spend about a minute max on each url. I don't want to make it wait much more. I believe the test will time out if we make it too long.
wait: 5s, 5s, 5s, 5s, 5s, 5s, 5s, 5s, 5s

I suggest we try this first. WDYT? Here are the tenacity docs:
https://tenacity.readthedocs.io/en/latest/

Reply:
Sounds good.